configure.ac

```diff
@@ -74,9 +74,15 @@ AC_CHECK_HEADER(thenonexistentheader.h, HAVE_THENONEXISTENTHEADER_H="true")
 
 # set some ld -R nonsense
 #
+AC_ARG_ENABLE(rpath,
+[  --disable-rpath         do not hardcode runtime library paths],
+:, enable_rpath=yes)
```

Comment:
Let's reply inline.

> There is two comas, so enable_rpath=yes it's action-if-not-given and this is the default state.

I never contested that. But since you specified as first argument `rpath`, the option will be `--enable-rpath`. And therefore that `action-if-not-given` clause will be active *iff `--enable-rpath` was not given*.